### DISCUSSION TITLE
Number of cpu cores available

### DISCUSSION THREAD

**Discussion by btessore** [whitphx/stlite#1015] [Q&A]
Hi everyone !

tl; dr: stlite app.exe works well, however uses only 1 cpu which makes it run slow

I am in the process of making a .exe application that runs deep learning models (using opencv dnn module). However, 
while with a pure streamlit use in a web browser the inference of my model seems to properly use all available cpu cores, in the stlite app, the python command os.cpu_count() returns 1 and indeed, the inference is very slow. 
I was wondering if it is a bug, if it is something expected or I I haven't configured my package.json proplerly. 

I thank you in advance for your help,
cheers,
B.

**Comment by whitphx** [comment]
Unfortunately it's Pyodide's nature and alaik there is no workaround for it now.
Pyodide executes Python code in the single-threaded browser environment where multi-threading/multi-processing are not available.
